storops_test/unity/resource/test_lun.py

```diff
@@ -525,6 +526,35 @@ def test_replicate_with_dst_resource_provisioning(self,
         assert_that(rep_session.id, equal_to(
             '42949675780_FNM00150600267_0000_42949678642_FNM00152000052_0000'))
 
+    @patch_rest
+    def test_is_vmware_vmfs_false(self):
+        vmfs = UnityLun.get(cli=t_rest(), _id='sv_4')
+        assert_that(vmfs.is_vmware_vmfs, equal_to(False))
+
+    @patch_rest
+    def test_is_vmware_vmfs_true(self):
+        vmfs = UnityLun.get(cli=t_rest(), _id='sv_5613')
+        assert_that(vmfs.is_vmware_vmfs, equal_to(True))
+
+    @patch_rest
+    def test_modify_vmfs_name(self):
```

Comment:
No modify step in this case

Reply:
Removing this case. Renaming is covered by `test_modify_vmfs_major_version_block_size`.